### PR TITLE
[ROCm] Bring up clang support for JAX+XLA

### PR DESCRIPTION
**Pull request by Ruturaj4**
* Add clang path

* bazelrc env fixes

* Fix wheelhouse installation and preserve wheels

* dockerfile changes

* Add target.lst

* Change target architectures

* Install bzip2 and sqlite packages

### CODE REVIEW

File: build/rocm/ci_build [hunk 1]
```diff
@@ -130,7 +130,7 @@ def _fetch_jax_metadata(xla_path):
 
     cmd = ["python3", "setup.py", "-V"]
     env = dict(os.environ)
-    env["JAX_RELEASE"] = "1"
+    #env["JAX_RELEASE"] = "1"
```
Comment (dfm) [inline build/rocm/ci_build:133]: nit: remove instead of commenting?

Comment (Ruturaj4) [reply build/rocm/ci_build]: Oops actually we need this :-) Thanks.

File: build/rocm/Dockerfile.ms [hunk 2]
```diff
@@ -70,14 +76,23 @@ FROM rocm_base AS rt_build
...
+# Create a directory to copy and retain the wheels in the image.
+RUN mkdir -p /rocm_jax_wheels
+
 RUN --mount=type=cache,mode=0755,target=/root/.cache/pip \
     --mount=type=bind,source=wheelhouse,target=/wheelhouse \
-    pip install --find-links /wheelhouse jax jaxlib jax_rocm60_plugin jax_rocm60_pjrt
+    cp /wheelhouse/* /rocm_jax_wheels/ && \
+    ls -lah /wheelhouse && \
+    pip3 install wheelhouse/*none*.whl wheelhouse/*jaxlib*.whl && \
+    pip3 install wheelhouse/*rocm60*.whl
```
Comment (JehandadKhan) [inline build/rocm/Dockerfile.ms:96]: [anchor]

**Review by JehandadKhan — APPROVED** [review]
LGTM

**Comment by rafrafek** [comment]
I think gfx1101 (RX 7800 XT 16 GB) is also supported by ROCm on Windows. Is it possible to add it here?

**Review by dfm — APPROVED** [review]

**Pull request merged**